### ISSUE TITLE
about Zero-center by mean pixel

### ISSUE THREAD

**Issue by joffery** [rcmalli/keras-vggface#10]
# TF order aka 'channel-last'
x = x[:, :, :, ::-1]
# TH order aka 'channel-first'
# x = x[:, ::-1, :, :]
# Zero-center by mean pixel
x[:, 0, :, :] -= 93.5940
x[:, 1, :, :] -= 104.7624
x[:, 2, :, :] -= 129.1863

the last three rows shouldn't be the following?:
x[:, : , :, 0] -= 93.5940
x[:, : , :, 1] -= 104.7624
x[:, : , :, 2] -= 129.1863
it makes no sense to do zero-center for the first three rows of an image.

**Comment by monktastic** [comment]
Yes, this looks correct for tf. The code given was for theano ordering (channels_first).
